token.go

```diff
@@ -447,6 +448,18 @@ func parseLoginAck(r *tdsBuffer) loginAckStruct {
 	return res
 }
 
+func parseFeatureExtAck(r *tdsBuffer) {
+	for {
```

Comment:
I don't like unbounded loops when parsing inputs.

Can we put some ridiculous large constraint here like 1000 or even just 10?
for i := 0; i < 10; i++ {
   ...
}
panic("parsed more then X and we are worried!")

Reply:
I think that's fair. I'll go back to the spec and see if there is a maximum number of features that is accepted, most likely each possible feature can only be present once in the login packet, so only once in the login response.